misc/scraper/localist_fetch.py

```diff
@@ -0,0 +1,122 @@
+import json
+
+import requests
+
+if __name__ == '__main__':
+    MAX_EVENT_PER_PAGE = 100
+    SCRAPER_START_DATE = "2019-10-30"
+    SCRAPER_END_DATE = "2019-10-31"
+
+    URL = "https://events.cornell.edu/api/2/events?start={0}&end={1}&pp={2}".format(SCRAPER_START_DATE,
```

Comment:
Nits: Using python3's format string can make it more readable:
```python
f"https://events.cornell.edu/api/2/events?start={SCRAPER_END_DATE}&end={SCRAPER_START_DATE}&pp={MAX_EVENT_PER_PAGE}"
```

Reply:
Ooh thanks for the heads up!